nabs/callbacks.py

```diff
@@ -49,31 +51,9 @@ class ELogPoster(CallbackBase):
                   'number': '{:.3g}',
                   'integer': '{:d}'}
 
-    _html_head = """<!DOCTYPE html>
-<html>
-<head>
-<style>
-table {
-  font-family: arial, sans-serif;
-  border-collapse: collapse;
-  width: 100%;
-}
-
-td, th {
-  border: 1px solid #dddddd;
-  text-align: left;
-  padding: 8px;
-}
-
-tr:nth-child(even) {
-  background-color: #dddddd;
-}
-</style>
-</head>
-<body>
-"""
+    _html_head = run_table_head
```

Comment:
Side thought: we might also consider adding jinja templates in the mix one day to make formatting these things a bit easier.